Content.Server/GameObjects/Components/Mobs/Body/Prototypes/LimbPrototype.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using YamlDotNet.RepresentationModel;
+using Robust.Shared.Serialization;
+using Robust.Shared.Prototypes;
+using Robust.Shared.IoC;
+using Robust.Shared.Utility;
+using Robust.Shared.Interfaces.Reflection;
+using Content.Server.GameObjects.Components.Mobs.Body.Organs;
+
+namespace Content.Server.GameObjects.Components.Mobs.Body
+{
+    /// <summary>
+    ///     Limb is not just like <see cref="Organ"/>, it has BONES, and holds organs (and child limbs), 
+    ///     it receive damage first, then through resistances and such it transfers the damage to organs,
+    ///     also the limb is visible, and it can be targeted
+    /// </summary>
+    [Prototype("limb")]
+    public class LimbPrototype : IPrototype, IIndexedPrototype
+    {
+#pragma warning disable CS0649
+        [Dependency]
+        protected IReflectionManager reflectionManager;
+#pragma warning restore
+
+        string IIndexedPrototype.ID => Id;
+
+        public string Name { get; private set; }
+        public string Id { get; private set; }
+        private string ClassName = "Limb";
```

Comment:
Why can't I just custom classes for `Limb` instances like I can with organs?

Reply:
well you can code it once you'll actually need it